subjects/own-and-return/README.md

```diff
@@ -0,0 +1,42 @@
+## own_and_return
+
+### Instructions
+
+Create a function `only_return` that takes a `struct` Film that has one Field of type `String` and returns the string without owning it.
+
+Create a function `own_and_return` which takes ownership of the struct, and returns the string.
+
+### Expected functions
+
+```rust
+pub struct Film {
+    pub name: String,
+}
+
+pub fn only_return(your_parameter) {
+}
+
+pub fn own_and_return(your_parameter) {
+}
+
+
+```
+
+### Usage
+
+Here is a possible program to test your function :
+
+```rust
```

Comment:
I've initially tough not to give `Usage` for this exercise, still I think you are right and we may give one, but the actual one is not a working program and do not follow our conventions about unimplemented code.

What do you think about this:
```rust
use own_and_return::*;

pub struct Film {
    pub name: String,
}

fn main() {
    let my_film = Film { name: "Terminator" };

    println!("{}", only_return(/* to be implemented */));
    println!("{}", only_return(/* to be implemented */));

    println!("{}", own_and_return(/* to be implemented */));

    // If any of the two following lines is uncommented the program should not compile.
    //  println!("{}",  only_return(\* to be implemented *\));
    //  println!("{}",  own_and_return(\* to be implemented *\));
}
```

Then you can also give an expected output:
And its output:

```console
$ cargo run
Terminator
Terminator
Terminator
$
```

Reply:
Thanks. Done

Reply:
The main function do not correspond to the output, also the commented part is missing (this is an important hint for the student and it gives a way for them to test everything is working as intended)

Reply:
I added some comments explaining how to test it. It's a bit tricky because of the problem of how to test something that doesn't compile, at  running time :slightly_smiling_face: . I fixed the rest.